### PR TITLE
Az4425-doc-collection2

### PR DESCRIPTION
**Pull request by Michael-McAuley-SSC**
Ovi will be adding more sections to this document afterwards.

### CODE REVIEW

**Review by davelanglois-ssc — APPROVED** [review]
Terrific work @Michael-McAuley-SSC  !!!

**Review by ovidiuasiminei-ssc — APPROVED** [review]
Great work Michael !

File: Architecture/Addendum.md [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+
+#### DNS Zones
+
+#### Internal DNS
+
+Internal DNS names are names that Google Cloud creates automatically. Available in a single VPC by default, resolved by the metadata server (169.254.169.254). Google Cloud automatically creates, updates, and removes these DNS records. Please see [Internal DNS zone documentation](https://cloud.google.com/compute/docs/internal-dns)
+
+#### Private Zones
+ 
+Managed custom domain names for virtual machines and other GCP resources without exposing the DNS data to the public internet. Please see [Private Zone Documentation]()
```
Comment (stephanemillaire-ssc) [inline Architecture/Addendum.md:77]: missing punctuation

File: Architecture/Addendum.md [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+
+[Cloud DNS Overview](https://cloud.google.com/dns/docs/overview/)
+
+[Cloud Domains Overview](https://cloud.google.com/domains/docs/overview)  
+
+#### DNS Zones
+
+#### Internal DNS
+
+Internal DNS names are names that Google Cloud creates automatically. Available in a single VPC by default, resolved by the metadata server (169.254.169.254). Google Cloud automatically creates, updates, and removes these DNS records. Please see [Internal DNS zone documentation](https://cloud.google.com/compute/docs/internal-dns)
```
Comment (stephanemillaire-ssc) [inline Architecture/Addendum.md:73]: missing punctuation

File: Architecture/Addendum.md [hunk 3]
```diff
@@ -0,0 +1,188 @@
...
+The ultimate [infographic on GCP products](https://googlecloudcheatsheet.withgoogle.com/) for Developers 
+
+Check you regions latency using [GPCping](https://gcping.com/)
+
+
+### DNS
+
+#### [DNS Overview](https://cloud.google.com/dns/docs/dns-overview)
+
+High level overview of some of the Google Cloud resource types to accelerate the design and deployment of your *workload*
```
Comment (stephanemillaire-ssc) [inline Architecture/Addendum.md:51]: add punctuation

File: Architecture/Addendum.md [hunk 4]
```diff
@@ -0,0 +1,188 @@
...
+Getting started with [Config Sync](https://cloud.google.com/anthos-config-management/docs/tutorials/config-sync-multi-repo)
+
+#### Tools and Utilities
+
+[Nomos](https://cloud.google.com/anthos-config-management/docs/how-to/nomos-command)  
+[Kubectl](https://cloud.google.com/anthos-config-management/docs/how-to/configure-config-sync-kubectl)  
+
+### kpt
+
+[KPT Readme.md](https://github.com/GoogleContainerTools/kpt/#readme) is an open source project used to hydrate yaml, get packages, apply functions, search and replace in yaml manifests. kpt is a package centric toolchain that enables a WYSIWYG configuration authoring and authoring enabling tool
```
Comment (stephanemillaire-ssc) [inline Architecture/Addendum.md:182]: add punctuation

**Review by stephanemillaire-ssc — COMMENTED** [review]
this will be tremendously useful...awesome work!

File: Architecture/Addendum.md [hunk 5]
```diff
@@ -0,0 +1,188 @@
...
+
+--------------------------------------
+
+### Fun Links
+
+Enjoy!
+
+The ultimate [infographic on GCP products](https://googlecloudcheatsheet.withgoogle.com/) for Developers 
+
+Check you regions latency using [GPCping](https://gcping.com/)
```
Comment (lucstjean-ssc) [inline Architecture/Addendum.md:44]: Should be "check your regions" and not "you regions"

**Comment by lucstjean-ssc** [comment]
Great doc!

File: Architecture/Addendum.md [hunk 6]
```diff
@@ -0,0 +1,188 @@
...
+### Cloud Armor
+
+#### [Cloud Armor Overview ](https://cloud.google.com/armor/docs/cloud-armor-overview)
+
+Google Cloud Armor is a DDoS and application defense service the helps protect your *service*. Cloud Armor is tightly coupled with the Global HTTPS(S) Cloud Load Balancer. Cloud Armor protects your network services, typically behind a load balancer from DOS and other web based attacks. Enforcement is managed at the edge Point of Presence (POP), as close to the source traffic as possible. Traffic can be filtered and defended against at layer 7 of the OSI model. 
+
+##### Cloud Armour Encompasses:
+
+* Policies/Rules
+* Whitelist / Blacklist IP's
```
[the remark this answers is not in the document]
Comment (Michael-McAuley-SSC) [reply Architecture/Addendum.md]: updated...

File: Architecture/Addendum.md [hunk 7]
```diff
@@ -0,0 +1,188 @@
...
+* Allows edge traffic blocking.  
+
+####  Cloud Armor Documentation of Interest
+
+[Use cases](https://cloud.google.com/armor/docs/common-use-cases)  
+[Security Policies](https://cloud.google.com/armor/docs/configure-security-policies#https-load-balancer)  
+[Rules Language Reference](https://cloud.google.com/armor/docs/rules-language-reference)  
+[Cloud Armor Standard VS Managed Protection Plus](https://cloud.google.com/armor/docs/managed-protection-overview#standard_versus_plus)  
+
+### Vpc Service Controls
```
[the remark this answers is not in the document]
Comment (Michael-McAuley-SSC) [reply Architecture/Addendum.md]: Fixed.

File: Architecture/Addendum.md (hunk 5 above)
Comment (Michael-McAuley-SSC) [reply Architecture/Addendum.md]: fixed

**Review by stephanemillaire-ssc — COMMENTED** [review]
lgtm

**Review by lucstjean-ssc — APPROVED** [review]

**Review by ovidiuasiminei-ssc — APPROVED** [review]
/lgtm

**Pull request merged**